### DISCUSSION TITLE
Float values shown along y-axis for integer entity

### DISCUSSION THREAD

**Discussion by GizmoGoody** [RomRider/apexcharts-card#475] [Q&A]
In this screenshot, the Phases y-axis should depict integer values instead of floats. The template entity is set as int and Developer Tools correctly reflects the current value as an integer. I've tried to set that series to 'float_precision: 0,' but that did not change the y-axis.

![image](https://user-images.githubusercontent.com/46235745/212103955-e2d38e24-6ad1-4c3f-85af-a6fe90cb6231.png)

Interestingly, the Current and Voltage template entities are floats, but their y-axis depicts integers increments, but I don't mind that.

**Comment by Kertz1954** [comment]
Try using a formatter in your yaxis. Remember to set a yaxis_id in your series to match. 

Example code.
```
  yaxis:
    - id: myidname
      show: true
      apex_config:
        labels:
          show: true
          formatter: |
            EVAL:function(value) {
            let result = parseFloat(value).toFixed(0);
            return result;
            }
  series:
    - entity: sensor.daily_gas_consumption_costs
      yaxis_id: myidname```
```

**Reply by GizmoGoody** [reply]
Thank you. That did the trick, but should it be necessary? Also, why would the other y-axis values (which are floats) depict as integers?
